src/hooks/useGameData.js

```diff
@@ -1,13 +1,13 @@
 import {useState} from 'react'
 import { v4 as uuid } from 'uuid'
-import saveLocally from '../lib/saveLocally'
-import loadlocally from '../lib/loadLocally'
+import SaveLocally from '../lib/SaveLocally'
```

Comment:
Should both be lowercase as they are no components.

Reply:
Filename and function name should be lowercase.

Reply:
That was a misunderstanding for my part and makes definitively more sense. :) I found the "l" what you meant. Thanks!